avae/loss.py

```diff
@@ -90,24 +90,24 @@ class AVAELoss:
 
     """
 
-    def __init__(self, device, beta, gamma=1, lookup_aff=None, recon_fn="MSE"):
+    def __init__(self, device, beta, gamma, lookup_aff=None, recon_fn="MSE"):
```

Comment:
Will this not break if there's no gamma?

Reply:
Do you mean no gamma in the input file ?

Reply:
Ah, because it's an array now, gotcha. I was just confused as gamma was optional as you don't need it to run vanilla beta-VAE.